examples/29-cluster-with-karpenter.yaml

```diff
@@ -12,7 +12,7 @@ iam:
   withOIDC: true
 
 karpenter:
-  version: 'v0.18.0' # Exact version must be provided
+  version: 'v0.20.0' # Exact version must be provided
   createServiceAccount: true # default is false
```

Comment:
```suggestion
  version: 'v0.20.0' # Exact version must be provided
  createServiceAccount: true # default is false
  withSpotInterruptionQueue: true # adds all required policies and rules for supporting Spot Interruption Queue, default is false
```

Reply:
The version here `integration/tests/karpenter/testdata/cluster-config.yaml` needs to be bumped to `0.20.0`

Reply:
https://github.com/weaveworks/eksctl/pull/6460/commits/8e4b1b7fa25cd34085c650de72f101eb6096f29a